### PR TITLE
task: some bugfixs

### PR DESCRIPTION
**Pull request by Burning1020**
1. report more OCI runtime errors
2. delete pid file of exec process
3. implement `close_io`

### CODE REVIEW

**Review by morningtzh — APPROVED** [review]
LGTM

File: vmm/task/src/container.rs [hunk 1]
```diff
@@ -245,28 +245,44 @@ impl KuasarFactory {
...
+    // Use match because `await` is not allowed inside lambda
+    match File::open(log_path).await {
+        Err(e) => Err(other!("unable to open OCI runtime log file: {}", e)),
         Ok(file) => {
             let mut lines = BufReader::new(file).lines();
             while let Ok(Some(line)) = lines.next_line().await {
                 // Retrieve the last runtime error
                 match serde_json::from_str::<Log>(&line) {
-                    Err(err) => return other!("{}: unable to parse log msg: {}", msg, err),
+                    Err(e) => return Err(other!("unable to parse log msg: {}", e)),
```
Comment (flyflypeng) [inline vmm/task/src/container.rs:277]: I suggest we can search for the most recent error-level log information in the log file by moving backward from the end of the file

Comment (Burning1020) [reply vmm/task/src/container.rs]: done

**Comment by flyflypeng** [comment]
LGTM

**Review by flyflypeng — APPROVED** [review]

**Pull request merged**